R/check_export_options.R

```diff
@@ -5,6 +5,7 @@
 #'
 #' @param dat secuTrialdata object
 #' @export
```

Comment:
does this even need exporting?

Reply:
Probably not I agree. If you really really want to run it you can always `secuTrialR:::`.

Reply:
Just as a side-note: this function is mentioned in the README. So if you decide to not export it, remember to adapt the README accordingly.